src/lib/cache.js

```diff
@@ -19,6 +20,16 @@ const isTest = NODE_ENV === "test"
 
 const VerboseEvents = ["issue", "failure", "reconnecting", "reconnect", "remove"]
 
+const cacheKeyDelimiter = "::"
+
+export const cacheKey = (key) => {
+  if (CACHE_COMPRESSION_ENABLED) {
+    return "1" + cacheKeyDelimiter + key
```

Comment:
@joeyAghion @alloy should I change this line to `return key` so as to preserve the existing compressed cache keys?  Otherwise we are going to effectively clear the cache when this is deployed.

Reply:
Good call 👍